src/envo/envo-edit.owl

```diff
@@ -40289,6 +40295,92 @@ AnnotationAssertion(<http://www.geneontology.org/formats/oboInOwl#inSubset> <htt
 AnnotationAssertion(rdfs:label <http://purl.obolibrary.org/obo/ENVO_06105112> "synthetic plastic"@en)
 SubClassOf(<http://purl.obolibrary.org/obo/ENVO_06105112> <http://purl.obolibrary.org/obo/ENVO_06105101>)
 
+# Class: <http://purl.obolibrary.org/obo/ENVO_06105113> (fishing)
+
+AnnotationAssertion(Annotation(<http://www.geneontology.org/formats/oboInOwl#hasDbXref> <https://en.wikipedia.org/wiki/Fishing>) Annotation(<http://www.geneontology.org/formats/oboInOwl#hasDbXref> <https://www.lexico.com/definition/fishing>) <http://purl.obolibrary.org/obo/IAO_0000115> <http://purl.obolibrary.org/obo/ENVO_06105113> "A planned environmental usage process during which any number of fish are caught by humans or their technologies."@en)
```

Comment:
German interpretations could include "fischen" and "angeln" (in case you'd like to add some multilingual support)